zipkin-server/src/main/resources/zipkin-server-shared.yml

```diff
@@ -144,6 +146,8 @@ zipkin:
       index-fetch-multiplier: ${CASSANDRA_INDEX_FETCH_MULTIPLIER:3}
       # Using ssl for connection, rely on Keystore
       use-ssl: ${CASSANDRA_USE_SSL:false}
+      # Override hostname verification in Cassandra setting
+      override-hostname-verification: ${CASSANDRA_OVERRIDE_HOSTNAME:false}
```

Comment:
I'd rather call the var `CASSANDRA_OVERRIDE_HOSTNAME_VERIFICATION`

Reply:
updated the variable name as suggested.

Reply:
for future, any time we are going to add a new variable, follow existing conventions first. The easiest way and regardless of cassandra or not, is to do a web or github action search for the property and use that. If there are multiple styles used, prefer the most used, for example other tools that configure cassandra.

Regardless, we don't want any incoherence in our codebase, so we needn't invent anything and just reuse the norm. My follow-up PR does that, just explaining here as I noticed sometimes folks accidentally don't look and then ask others to use non-conventional names. https://github.com/apache/cassandra-java-driver/blob/8d5849cb38995b312f29314d18256c0c3e94cf07/core/src/main/java/com/datastax/oss/driver/api/core/config/TypedDriverOption.java#L230